verifact/forum/fixtures/answers.json

```diff
@@ -8,8 +8,6 @@
           "text": "It's a bit more nuanced than that - blablabla",
```

Comment:
I actually think these files are no longer being used (correct me if I'm wrong). I replaced them with much better factories that are more flexible and concise to use. Assuming that's correct, you can delete these files.